### ISSUE TITLE
file change notify, like github.com/fsnotify/fsnotify?

### ISSUE THREAD

**Issue by corentone** [spf13/afero#131]
Hello!

In our infrastructure, we regularly refresh a config file and need to detect file changes in order to properly reload. I've been a regular user of afero.Fs and was looking into the best way to do it...

Would you have suggestions? We currently use both the regular OS Fs and the in-Mem Fs.

Thanks!

**Comment by purpleidea** [comment]
Great idea :)

**Comment by mbertschler** [comment]
How would you like to use that feature? We could use the fsnotify for the standard OS filesystem but for all the other use cases we would need to implement it in the MemMapFs and so on.

**Comment by purpleidea** [comment]
@mbertschler Of course, although while OS would use fnotify, MemMapFs could also implement this by easily adding a hook onto any chmod or write call to any of the files. Similar patterns exist for the other backends.

**Comment by akuzia** [comment]
Any progress on this feature?
